style.css

```
@@ -39,3 +77,98 @@ p {
   box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
   background: white;
 }
+
+/* 대기중 개수 */
+.wait {
+  padding: 15px 15px 5px 15px;
+  font-weight: bold;
+  font-size: 17px;
+}
+
+/* 완료 개수 */
+.complete {
+  padding: 15px 15px 5px 15px;
+  font-weight: bold;
+  font-size: 17px;
+}
+
+/* 완료 시 선긋기 */
+#completeList {
+  text-decoration: line-through;
+  color: #a7a7a7;
+}
+
+/* todo 목록 */
+.list {
+  height: 10px;     /* list div의 크기 고정 위해 */
+  flex-grow: 8;
+  padding: 10px 10px;
+
+  overflow: scroll;
+  overflow-x: hidden;
+
+  /* inner items */
+  display: flex;
+  flex-direction: column;
+}
+
+/* 할 일 추가 */
+.add {
+  display: inline-flex;
```

혹시 입력 칸 밑에 무엇가가 추가되었을 때 inline요소를 가지고 있으면 컨테이너가 가로로 배치된다고 하네요!! 그렇게 된다면 지금 잡아 놓은 css가 무너질수도 있으니 안전성을 위해 flex로 지정하면 좋을 거 같아요! 코드 길이도 짧아지고..! 단지 제 의견입니다!! 
```suggestion
  display: flex;
```

---

와우 최고에요

---

input과 button 이 가로로 정렬되어야 해서 input에 inline-flex를 적용하였는데,
그냥 flex를 적용해도 되는거였나요 ?